api/envoy/config/filter/http/gzip/v2/gzip.proto

```diff
@@ -11,6 +11,7 @@ import "google/protobuf/wrappers.proto";
 
 import "validate/validate.proto";
 import "gogoproto/gogo.proto";
+import "envoy/config/filter/http/compressor/v2/compressor.proto";
```

Comment:
Can you move this up to its own block, we tend to have the Envoy specific imports at a further up in the file.